config/config.ts

```diff
@@ -0,0 +1,27 @@
+export class Config  {
+    #accountID: string = String(process.env.ACCOUNTID);
+    #userName: string = String(process.env.USERNAME);
+    #password: string = String(process.env.PASSWORD);
+    #url: string = process.env.URL ? String(process.env.URL) : "https://skyvegas.com.nxt.ppbdev.com";
```

Comment:
are these string casts necessary?
what about this #s in front of every name, is that some ts or playwright standard? never seen it before

Reply:
Good to get all this sorted into a config though. Can load it once at start, and extend it as we need new stuff e.g. additional URLs